tool/settings.py

```diff
@@ -0,0 +1,121 @@
+"""
+Django settings for tool project.
+
+Generated by 'django-admin startproject' using Django 1.11.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.11/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
```

Comment:
Add `STATIC_ROOT = os.path.join(BASE_DIR, 'static')` to avoid any errors while `collectstatic`.